File: GraffiTab_iOS_SDK/Classes/Managers/Feedback/GTFeedbackManager.m
//
//  GTFeedbackManager.m
//  GraffiTabiOSSDK
//
//  Created by Georgi Christov on 29/04/2015.
//  Copyright (c) 2015 GraffiTab. All rights reserved.
//

#import "GTFeedbackManager.h"
#import "GTSendFeedback.h"

@implementation GTFeedbackManager

+ (void)postFeedbackWithText:(NSString *)text successBlock:(void (^)(GTResponseObject *))successBlock failureBlock:(void (^)(GTResponseObject *))failureBlock {
    GTSendFeedback *task = [GTSendFeedback new];
    [task postFeedbackWithText:text successBlock:successBlock failureBlock:failureBlock];
}

@end

File: GraffiTab-iOS-SDK/Managers/Api/GTFeedbackManager.swift
//
//  GTFeedbackManager.swift
//  GraffiTab-iOS-SDK
//
//  Created by Georgi Christov on 13/04/2016.
//  Copyright © 2016 GraffiTab. All rights reserved.
//

import UIKit
import Alamofire

public class GTFeedbackManager: NSObject {

    public class func sendFeedback(name: String, email: String, text: String, successBlock: (response: GTResponseObject) -> Void, failureBlock: (response: GTResponseObject) -> Void) -> Request {
        let task = GTSendFeedbackTask()
        return task.sendFeedback(name, email: email, text: text, successBlock: successBlock, failureBlock: failureBlock)
    }
}
